### PR TITLE
Create new location

### PR DESCRIPTION
**Pull request by tangenta**

### CODE REVIEW

**Comment by nginx-training** [comment]
nice work

**Comment by wordchild** [comment]
I think you might have some extra spaces after colons and blank lines! (I had the same problem with mine)

File: new location [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (crichID) [inline new location]: @tangenta please remove this blank line from the beginning of your file!

**Comment by crichID** [comment]
@tangenta your file must be named `_pins/tangenta.json` in order for the checks to pass. If you need any help, please check out the following video: https://vimeo.com/201011288

**Comment by hollenberry** [comment]
@tangenta  I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/6606/commits/aa0cbffd8edc01e9e9398189f5698bdbf5e30711. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by May 26, I will go ahead and do it just to keep the repository clean!

**Pull request merged**